<<<<<<< HEAD
<<<<<<< HEAD
const crash = reason => {
  process.on(reason, err => {
=======
const crash = (reason) => {
  process.on(reason, (err) => {
>>>>>>> 3eff7c172e63406dbb70d3c6ccf8d675b3138252
=======
const crash = (reason) => {
  process.on(reason, (err) => {
>>>>>>> 60e4365cdd99277edacce2bf8c629e1668cfac86
    console.error(reason, err.stack)
    process.exit(-1)
  })
}

crash('unhandledRejection')
crash('uncaughtError')
crash('warning')
